extensions/cocostudio/armature/animation/CCArmatureAnimation.js

```diff
@@ -292,8 +292,11 @@ ccs.ArmatureAnimation = ccs.ProcessBase.extend(/** @lends ccs.ArmatureAnimation#
                 tween.play(movementBoneData, durationTo, durationTween, loop, tweenEasing);
                 tween.setProcessScale(this._processScale);
 
-                if (bone.getChildArmature())
+                if (bone.getChildArmature()) {
```

Comment:
Hi @mutoo ,

Could you please give us more information about why did the modify, and how to reproduce the error or bug.

Thanks
David